src/blur.js

```diff
@@ -0,0 +1,88 @@
+import {floor, max, min} from "./math.js";
```

Comment:
Minor simplification opportunity: remove `src/math.js` and change this to:

```suggestion
const {floor, max, min} = Math;
```

Reply:
Yep, I'm not sure why in D3 some math functions are abstracted like this, and others are not.